src/moonstone-samples/lib/TimePickerSample.js

```diff
@@ -83,7 +84,7 @@ module.exports = kind({
 	},
 	timeChanged: function (sender, event) {
 		if (this.$.result && event.value){
-			var timeArray = event.value.toTimeString().split(': ');
+			var timeArray = event.value.toTimeString().split(':');
```

Comment:
I don't know if we need to split the time string anymore - can we just display the whole time string, instead?